common/on_actions/imperia_on_monthtly_pulse.txt

```diff
@@ -271,46 +271,19 @@
 			remove_variable ?= ai_disabled_timer
 			set_variable = detected_player
 		}
-		# Demolish non-capital statues
 		if = {
 			limit = {
-				any_scope_building = {
-					is_building_type = building_power_bloc_statue
-					state = {
-						is_capital = no
-					}
-				}
-			}
-			every_scope_state = {
-				limit = {
-					has_building = building_power_bloc_statue
-					is_capital = no
+				NOT = {
+					has_global_variable = current_max_multiculturalism_level
 				}
-				remove_building = building_power_bloc_statue
 			}
-		}
```

Comment:
This is part of the "only statues in the capital" change that Fallonar wanted, shouldnt be removed. Unless I'm misunderstanding how Github presents it?